types/fparser/fparser.d.ts

```diff
@@ -0,0 +1,43 @@
+declare class FormulaC {
```

Comment:
```suggestion
export default class Formula {
```
and move this file into `./src/` subdirectory (it's defined in `package.json` ESM support